### ISSUE TITLE
Any plans to make Timoni usable as a library

### ISSUE THREAD

**Issue by m13t** [stefanprodan/timoni#170]
Appreciate it’s early days with this project so settling on an API interface at this point may a bit eager, but do you have plans to allow Timoni to be used as a library in other Go projects?

**Comment by stefanprodan** [comment]
Yes for sure, in the future hopefully we'll have a Timoni Kubernetes controller that will use the same library as the CLI. What's now in the `internal` package, will be moved to `pkg` once the API becomes more stable.

**Comment by rd-niklas-gergs** [comment]
Great tool :) Is there a rough ETA for the library functionality? We would especially be interested in using what corresponds to the `timoni build` command as a library calls.

**Comment by suse-coder** [comment]
I would also love to see this, where from a web ui one can call a api and then new timoni bundles/modules are downloaded and with fluxcd installed into the cluster
